### PR TITLE
Update client-go to Kubernetes v1.18

### PR DESCRIPTION
**Pull request by mikkeloscar**
Updates client-go (+ friends) to Kubernetes v1.18. Functionality stays the same.

### CODE REVIEW

File: controller/stack_resources_test.go [hunk 1]
```diff
@@ -650,23 +651,23 @@ func TestReconcileStackIngress(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			env := NewTestEnvironment()
 
-			err := env.CreateStacksets([]zv1.StackSet{testStackSet})
+			err := env.CreateStacksets(context.TODO(), []zv1.StackSet{testStackSet})
 			require.NoError(t, err)
 
-			err = env.CreateStacks([]zv1.Stack{tc.stack})
+			err = env.CreateStacks(context.TODO(), []zv1.Stack{tc.stack})
```
Comment (linki) [inline controller/stack_resources_test.go:657]: I think top-level contexts in tests are typically using Background: https://golang.org/pkg/context/#Background

Comment (mikkeloscar) [reply controller/stack_resources_test.go]: Changed to use `context.Background()`

**Comment by linki** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**